step1-07/demo/README.md

```diff
@@ -131,17 +133,17 @@ interface TodoListProps {
 
 Our `TodoApp` doesn't take any props, but it does have state. We can use TypeScript to define that as well.
 
-I've already imported `Todos`, and `FilterTypes` into the `TodoApp`, so we just need to add them to our class. We can even skip the 'interface', if we want to, and add them directly to the class.
+I've already imported `Todos` and `FilterTypes` into the `TodoApp`, so we just need to add them to our class. If we want, we can even skip a separate interface definition and just declare the type inline. (This is not recommended for types of any complexity or types that are used in multiple places.)
```

Comment:
It's probably worth calling out that it's possible to do type definitions inline, but I'd also like to mention that it's usually not recommended (due to potentially reducing readability and encouraging repetition)